shell/platform/embedder/tests/embedder_test_context_vulkan.h

```diff
@@ -33,6 +33,10 @@ class EmbedderTestContextVulkan : public EmbedderTestContext {
 
   bool PresentImage(VkImage image);
 
+  static void* InstanceProcAddr(void* user_data,
+                                FlutterVulkanInstanceHandle instance,
+                                const char* name);
```

Comment:
optional nitpick since this is private API: do we not use `std::string_view` elsewhere in this API? If not, then nothing to do here.

Reply:
It's not a private API, the function pointer's signature has to match the one we've declared in `embedder.h`.

Reply:
Ugh I see, yeah, needs to match the embedder.h declaration.